src/cim/components/helpers/toAssisted.ts

```diff
@@ -102,3 +103,13 @@ export const getAICluster = ({
 
   return aiCluster;
 };
+
+export const getOCPVersions = (
```

Comment:
This function would better fit into it's own module (`versions` maybe?). `toAssisted` is a module for utils converting k8s resources into REST API based resources.

Reply:
ok, moved to `versions`